optuna/integration/pytorch_distributed.py

```diff
@@ -36,7 +35,7 @@
     "Use :func:`~optuna.integration.TorchDistributedTrial.suggest_float` instead."
 )
 
-_g_pg: List[Optional["ProcessGroup"]] = [None]
+_g_pg: Optional["ProcessGroup"] = None
```

Comment:
This change seems reasonable to me since only the first element is used in the `master` code.
I'd like to use a local variable to manage the process group for Optuna in https://github.com/optuna/optuna/pull/4268, but I don't think we should work on it in this PR.

Reply:
Thank you for your follow-up.